docs/en/remote-flow.rst

```diff
@@ -457,7 +412,7 @@ Below is a non-normative example of the decrypted JSON ``response`` content:
         "id": "04a98be3-7fb0-4cf5-af9a-31579c8b0e7d",
         "descriptor_map": [
             {
-                "id": "eu.europa.ec.eudiw.pid.it.1",
+                "id": "PersonIdentificationData",
                 "path": "$.vp_token.verified_claims.claims._sd[0]",
```

Comment:
@fmarino-ipzs we don't have anymore hte verified_claims

Reply:
please align your PR with this
https://github.com/italia/eudi-wallet-it-docs/pull/312/files

Reply:
I would suggest to replace the actual _"path": "$.vp_token.verified_claims.claims._sd[0]_ with $.vp_token.verifiableCredential[0]
Please note that the actual format points to the _sd claim and not to the credential as whole.

Reply:
@pietroACN we don't have `verifiableCredential` in the current vp_token. We have an array where each entry is given below

{
  "iss": "vbeXJksM45xphtANnCiG6mCyuU4jfGNzopGuKvogg9c",
  "jti": "3978344f-8596-4c3a-a978-8fcaba3903c5",
  "aud": "https://relying-party.example.org/response_uri",
  "iat": 1541493724,
  "exp": 1573029723,
  "nonce": "2c128e4d-fc91-4cd3-86b8-18bdea0988cb",
  "vp": "<Issuer-Signed-JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>"
}

If I understand correctly, the right path should be `$.vp_token[0].vp`

Do you agree?

We will change this in the next milestone according to the latest version of the reference specification

Reply:
@fmarino-ipzs indeed the right path would be $.vp_token.<identifier>[n] where <identifier> defines the entry type and [n] identifies one of the entries (n-th).

In OpenID <identifier> is verifiableCredential, and out tokens are just representation of a Credential.